src/Storages/MergeTree/Compaction/PartsCollectors/MergeTreePartsCollector.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <Storages/MergeTree/Compaction/PartsCollectors/IPartsCollector.h>
```

Comment:
We don't have `Compaction` word anywhere, in clickhouse it's called `Merge` . But okay, we can leave it as is.

Reply:
Discussed in dm.